content/en/docs/tasks/observability/telemetry/index.md

```diff
@@ -61,21 +62,35 @@ data:
 
 For convenience, Istio comes with a few providers configured out of the box with default settings:
 
-| Provider Name | Functionality                    |
-| ------------- | -------------------------------- |
-| `prometheus`  | Metrics                          |
-| `stackdriver` | Metrics, Tracing, Access Logging |
-| `envoy`       | Access Logging                   |
+| Provider Name | Functionality              |
+| ------------- | -------------------------- |
+| `prometheus`  | Metrics                    |
+| `stackdriver` | Tracing (legacy / limited) |
+| `envoy`       | Access Logging             |
 
-In additional, a [default provider](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-DefaultProviders) can be set which
+{{< warning >}}
```

Comment:
This doesn't need to be a warning. This is an old change at this point. Leaving a note sure, but we don't need a warning box.